certbot-dns-digitalocean/certbot_dns_digitalocean/dns_digitalocean_test.py

```diff
@@ -2,6 +2,7 @@
 
 import os
 import unittest
+import six
```

Comment:
It seems like `six` should be added to `certbot-dns-digitalocean`'s `setup.py` file.